nifi-nar-bundles/nifi-media-bundle/nifi-media-processors/src/test/java/org/apache/nifi/processors/image/ExtractImageMetadataTest.java

```diff
@@ -37,7 +37,7 @@
     private static String BMP_HEADER = "BMP Header.";
     private static String JPEG_HEADER = "JPEG.";
     private static String GIF_HEADER = "GIF Header.";
-    private static String PNG_HEADER = "PNG.";
+    private static String PNG_HEADER = "PNG-";
```

Comment:
When composing Nar's people often think about how various processors do similar things and make sense as a bundle of like things from a user perspective.  However, they're really only about classloader isolation and so the thinking should be more purely about their dependencies and if they can/should co-exist nicely.  This case sounds like this to me where we're thinking intuitively that these processors should be bundled because they do similar (media processing) things and then we're trying to wrangle their dependencies.

This is just a food for thought comment.  I'm not suggesting we have to change this one.  It may be correct as is.

For this specific case, could we perhaps bridge the header/attribute used so that it will map from the new name to the old name?